File: httpclient-app/src/main/java/sk/httpclient/app/Record.java
package sk.httpclient.app;

import java.util.HashMap;
import java.util.Map;

public class Record implements Getable {
    private String name = "defaultName";
    private Integer age = 31;
    private String city = "default city";

    public Record() {
    }

    public Record(String name, Integer age, String city) {
        this.name = name;
        this.age = age;
        this.city = city;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getAge() {
        return age;
    }

    public void setAge(Integer age) {
        this.age = age;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    @Override
    public String toString() {
        return "Record{" +
                "name='" + name + '\'' +
                ", age=" + age +
                ", city='" + city + '\'' +
                '}';
    }

    @Override
    public Map<String, String> toMap() {
        Map<String, String> map = new HashMap<>();
        map.put("age", "132");
        return map;
    }
}

File: httpclient-app/src/main/scala/sk/http/app/Record.scala
package sk.http.app

import sk.httpclient.client.Getable
import java.util.HashMap
import java.util.Map

class Record() extends Getable {
  private var name: String = "defaultName"
  private var age: Integer = 31
  private var city: String = "default city"

  def this(name: String, age: Integer, city: String) {
    this()
    this.name = name
    this.age = age
    this.city = city
  }

  def getName: String = {
    return name
  }

  def setName(name: String) {
    this.name = name
  }

  def getAge: Integer = {
    return age
  }

  def setAge(age: Integer) {
    this.age = age
  }

  def getCity: String = {
    return city
  }

  def setCity(city: String) {
    this.city = city
  }

  override def toString: String = {
    return "Record{" + "name='" + name + '\'' + ", age=" + age + ", city='" + city + '\'' + '}'
  }

  def toMap: java.util.Map[String, String] = {
    val map: java.util.Map[String, String] = new java.util.HashMap[String, String]
    map.put("age", "" + this.age)
    return map
  }
}